week06/Convert_Sorted_Array_to_Binary_Search_Tree.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+TC : 0(N)   //where N is number of elements in nums
+MC : 0(N)
```

Comment:
It is O(H), where H is the depth of the tree. Recursion takes space.

NOT O(N).